botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql

```diff
@@ -0,0 +1,70 @@
+type Query {
+    botResponses(projectId: String!): [BotResponse!]
+    botResponse(projectId: String!, key: String!, lang: String): BotResponse
+    botResponseById(_id: String): BotResponse
+}
+
+type Subscription {
+    botResponseAdded(projectId: String!): BotResponse
+    botResponsesModified(projectId: String!): BotResponse
+    botResponseDeleted(projectId: String!): BotResponse
+}
+
+type Mutation {
+    deleteResponse(projectId: String!, key: String!): updateResponse!
+    updateResponse(
+        projectId: String!
+        _id: String!
+        response: BotResponseInput
+    ): updateResponse!
+    createResponse(projectId: String!, response: BotResponseInput): updateResponse!
+    createResponses(projectId: String!, responses: String): updateResponse!
+}
+
+type updateResponse {
+    success: Boolean!
+}
+
+type BotResponse {
+    key: String
+    _id: String
+    projectId: String
+    values: [Value]
+}
+
+type Value {
```

Comment:
I don't think we should name types with such a generic name.

Reply:
done